### DISCUSSION TITLE
WGS84 Area from polygon

### DISCUSSION THREAD

**Discussion by johnwc** [NetTopologySuite/NetTopologySuite#681] [Q&A]
Creating a polygon using WGS84(SRID:4326), what unit does the Area property return its value in? How can I convert it to square miles or square kilometers?

**Comment by KubaSzostak** [comment]
> Creating a polygon using WGS84(SRID:4326), what unit does the Area property return its value in? 

The area will be calculated assuming that the geometry is flat and will be expressed in degrees.

> How can I convert it to square miles or square kilometers?

It can be done but it's not an [easy task](https://journals.pan.pl/Content/98330/PDF/art3_PedzichKuzma.pdf). It would be hard to explain how to calculate it in a short comment. Keep in mind that there is no linear formula to convert the area. Lets assume we have a square with each side 1° long. The area of the square is `1° x 1° = 1°²`. The area expressed in square kilometres would depend on the square location. Near to equator it would be greater than near to pole. So, the square area will have 12308 km² somewhere in Africa and the same square would have 6122 km² somewhere in Norway.

![image](https://user-images.githubusercontent.com/4609437/233275903-cba35da4-b563-4504-baa0-33cf405f7a2f.png)

**Comment by KubaSzostak** [comment]
You can find some inspiration how to calculate an area on ellipsoid surface in my [Ellipsoidus](https://github.com/KubaSzostak/Ellipsoidus) project.
